### ISSUE TITLE
Testimonials mobile screen

### ISSUE THREAD

**Issue by santoshkpatro** [formester/formester-site#7]
testimonial page on mobile screen is not ok
![Screenshot_20211116-234246](https://user-images.githubusercontent.com/60396045/142045852-fd582cb9-0ac3-4823-9674-2abe35ddf596.jpg)

**Comment by piyushsinghania** [comment]
Yes @santoshkpatro we are already working on a new testimonials section it will be live soon. Thanks for posting!
